axum-extra/src/extract/form.rs

```diff
@@ -67,7 +67,7 @@ where
         if req.method() == Method::GET {
             let query = req.uri().query().unwrap_or_default();
             let value = serde_html_form::from_str(query)
-                .map_err(FailedToDeserializeQueryString::__private_new)?;
+                .map_err(FailedToDeserializeQueryString::__private_new::<(), _>)?;
```

Comment:
Actually, why is this `()` instead of T?

Reply:
This means that assuming this is released, and the new axum-extra is used with an older version of axum, the message will still include "for type _", but `_` will be `()`.

I guess we can release the new axum 0.5.x release, then update axum-extra's dependency so it has a matching minimum version, and release that 🤔

Reply:
> I guess we can release the new axum 0.5.x release, then update axum-extra's dependency so it has a matching minimum version, and release that

Yeah that was my plan.